src/ucp/api/ucp.h

```diff
@@ -679,6 +771,15 @@ typedef struct ucp_ep_params {
      * Handler to process transport level failure.
      */
     ucp_err_handler_t       err_handler;
+
+    /**
+     * Destination address in the form of a sockaddr; if a client-server connection
+     * establishment mechanism is used, this address must be filled and obtained
+     * from the user.
+     * The UCP_EP_PARAM_FIELD_SOCK_ADDR bit in the field_mask should be set
+     * to indicate that the type of the remote address is a sockaddr.
+     */
+    ucp_addr_sock_addr_t    sock_address;
```

Comment:
So, in terms of initialization of endpoint we can pass ucp_addr_sock_addr_t or ucp_address_t but not the both ?
If this is the case, it is very confusing.